### PR TITLE
Dm 4043 remove text and images component

### PR DESCRIPTION
**Pull request by PhilipDeFraties**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-4043?focusedCommentId=11762&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-11762

## Description - what does this code do?
removes "Text and Images" component from pb form along with corresponding feature spec

TODO: remove from DB and page view - DONE*

## Testing done - how did you test it/steps on how can another person can test it 
1. Create a page
2. Verify the "Text and Images" component is not selectable to add to the page

## Screenshots, Gifs, Videos from application (if applicable)
Screenshot 2023-07-19 at 4 37 02 PM

## Acceptance criteria
- [ ]

## Definition of done
- [x] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating JIRA issue
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by camillevilla** [comment]
✅ I removed Text and Images components from all of the environments just in case

File: app/assets/javascripts/active_admin.js [hunk 1]
```diff
@@ -329,76 +328,6 @@ const pageComponentNames = [
...
-                $lastAttrLiSelector.css('padding-bottom', '36px');
-            }
-        });
-    }
-
-    function showPageComponentImagesHeaderOnAddLinkClick() {
-        $(document).on('click', '.add-another-page-component-image', function() {
-            // If there are is at least one visible 'page-component-image-li' elements, show the header if it's not visible already
-            const $pageComponentImageHeader = $(this).siblings('.page-component-images-header');
-            if ($(this).prev().children('.page-component-image-li').not('.display-none').length
```
Comment (camillevilla) [inline app/assets/javascripts/active_admin.js:385]: Let's also remove the related `.page-component-image` etc. classes that are in `active_admin.scss`

**Review by camillevilla — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/active_admin.js (hunk 1 above)
Comment (PhilipDeFraties) [reply app/assets/javascripts/active_admin.js:385]: good eye

**Review by camillevilla — APPROVED** [review]
✅ Looks good! Thanks for making those changes.

**Pull request merged**